cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `namespace` parameter is defined in the `cinema.urls` and `user.urls` configurations. The `include` function with a `namespace` requires the included URLconf to define an `app_name` attribute.